tools/convert-old-video-settings-file.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+"""Convert legacy video formats to generate gstreamer pipeline
+    Move old files, such as vidformat and gstreamer2 parameter files
+    to the user trash folder.
+"""
+
+import os
+
+def merge_gst_elements(elements: list):
+    """A simple helper to merge gstreamer elements in pipeline format
+
+    Args:
+        elements (list): List that contains gstreamer elements
+
+    Returns:
+        str: Final gstreamer pipeline
+    """
+
+    return ' ! '.join(elements)
+
+
+class Convert:
+    """Convert main class
+        Check file description for more information
+    """
+
+    HOME = os.path.expanduser('~')
+    TRASH = os.path.join(HOME, '.local/share/Trash/files')
+    vidformat_file = os.path.join(HOME, 'vidformat.param')
+    gstreamer2_file = os.path.join(HOME, 'gstreamer2.param')
+
+    pipeline_settings = {
```

Comment:
Settings are not really for the pipeline, right? they are for the device, what about:
```suggestion
    device_settings = {
```